src/App.tsx

```diff
@@ -30,6 +38,23 @@ const App: React.FC = () => {
       <GlobalStyle />
       <header>
         <a href="https://reactjs.org">Learn React</a>
+        <Box flex paddingHorizontal={3.4}>
+          <Product
+            image="
+            https://demo2.wpopal.com/auros/wp-content/uploads/2018/10/1-21-620x698.jpg"
+            secondImage="https://demo2.wpopal.com/auros/wp-content/uploads/2018/10/2-21-600x675.jpg"
+            productName="Mega 2 Seater Sofa"
+            price={160.71}
+            buttonText=" add to card"
```

Comment:
Don't add the space here. If you need the space, add it before the value of `{buttonText}` where you use it (in `Product.jsx`). 

```diff
- buttonText=" add to card"
+ buttonText="add to card" // in App
+ <p>Some {buttonText} spacing</p> // in Product
```